ai-quick-actions/register-tips.md

```diff
@@ -0,0 +1,48 @@
+# Register a model
+
+Table of Contents:
+
+- [Home](README.md)
+- [CLI](cli-tips.md)
+- [Policies](policies/README.md)
+- [Model Deployment](model-deployment-tips.md)
+- [Model Evaluation](evaluation-tips.md)
+- [Model Fine Tuning](fine-tuning-tips.md)
+
+## Upload model artifact to Object Storage
+
+AI Quick Actions (AQUA) supports user-provided models that can be deployed, fined-tuned and evaluated. You can now upload
+and test the models with the artifacts downloaded either from Hugging Face or from your own models created from scratch.
```

Comment:
I'd rephrase:
"You can now upload and test models with artifacts downloaded from model repositories like Hugging Face, etc. or from your own models"

Reply:
updated.